test/test-signal.c

```diff
@@ -19,13 +19,37 @@
  * IN THE SOFTWARE.
  */
 
-
-/* This test does not pretend to be cross-platform. */
-#ifndef _WIN32
-
 #include "uv.h"
 #include "task.h"
 
+/* For Windows we test only signum handling */
+#ifdef _WIN32
+static void signal_cb(uv_signal_t* handle, int signum) { }
+
+TEST_IMPL(win32_signum_number) {
```

Comment:
This test is OK, but we need one which tests the whole reason behind this patch.  Can that be done?